samples/sensor/tmp108/src/main.c

```diff
@@ -142,8 +142,14 @@ void main(void)
 	tmp108 = DEVICE_DT_GET_ANY(ti_tmp108);
 
 	if (!tmp108) {
-		printf("error: no tmp108 device found\n");
-		return;
+		printf("warning: tmp108 device not found checking for compatible ams device\n");
+
+		tmp108 = DEVICE_DT_GET_ANY(ams_as6212);
```

Comment:
Since this is a sample I think is worth to be nit here, I suggest to rename tmp108 variable, it is strange to reuse it to gather another device with a different PN, even if they're compatible.

People who refer samples in general are searching a place to start, so keep code simpler to understand is always a good choice there.

something like `sensor_device` maybe? or declare two device descriptors, one to each sensor type.

Reply:
done